gammapy/estimators/excess_map.py

```diff
@@ -95,13 +97,15 @@ def __init__(
         selection_optional="all",
         energy_edges=None,
         apply_mask_fit=False,
+        correlate_off=False
```

Comment:
I would put the default at `True` since this is also the behavior for regular datasets.

Reply:
On the slack discussions, we discuss to by default not correlate off. As it seems to me that this function, in the case of a MapDatasetOnOff will mainly be called after a ring background it's seem to me better to not correlate off by default.

Reply:
OK. I suppose the important point is that it is well documented.